src/library/dlgtrackinfo.cpp

```diff
@@ -296,6 +321,7 @@ void DlgTrackInfo::loadTrackInternal(const TrackPointer& pTrack) {
     }
 
     m_pLoadedTrack = pTrack;
+    m_beatsChanged = false;
```

Comment:
Doesn't this belong into reloadTrackBeats()? Otherwise the flag is not reset when reloading the original values from the track!